src/inventory.rs

```diff
@@ -581,66 +642,85 @@ impl DeploymentInventory {
         println!("SSH user: {}", self.ssh_user);
         println!();
 
-        println!("============");
-        println!("Uploader VMs");
-        println!("============");
-        for vm in self.uploader_vms.iter() {
-            println!("{}: {}", vm.0, vm.1);
+        if !self.uploader_vms.is_empty() {
+            println!("============");
+            println!("Uploader VMs");
+            println!("============");
+            for vm in self.uploader_vms.iter() {
+                println!("{}: {}", vm.0, vm.1);
+            }
+            println!("SSH user: {}", self.ssh_user);
+            println!();
         }
-        println!("SSH user: {}", self.ssh_user);
-        println!();
 
-        println!("=========");
-        println!("Other VMs");
-        println!("=========");
-        for vm in self.misc_vms.iter() {
-            println!("{}: {}", vm.0, vm.1);
+        if !self.misc_vms.is_empty() {
+            println!("=========");
+            println!("Other VMs");
+            println!("=========");
+            for vm in self.misc_vms.iter() {
+                println!("{}: {}", vm.0, vm.1);
+            }
+            println!("SSH user: {}", self.ssh_user);
+            println!();
         }
-        println!("SSH user: {}", self.ssh_user);
-        println!();
 
-        // Take the first peer from each VM. If you just take, say, the first 10 on the peer list,
-        // they will all be from the same machine. They will be unique peers, but they won't look
-        // very random.
-        println!("============");
-        println!("Sample Peers");
-        println!("============");
-        self.bootstrap_node_vms
-            .iter()
-            .chain(self.node_vms.iter())
-            .map(|vm| vm.1.to_string())
-            .for_each(|ip| {
-                if let Some(peer) = self.peers().iter().find(|p| p.contains(&ip)) {
-                    println!("{peer}");
-                }
-            });
-        println!("Genesis: {}", self.genesis_multiaddr);
-        let inventory_file_path =
-            get_data_directory()?.join(format!("{}-inventory.json", self.name));
-        println!(
-            "The entire peer list can be found at {}",
-            inventory_file_path.to_string_lossy()
-        );
-        println!();
+        // If there are no bootstrap nodes, it's a bootstrap deploy, and in that case, we're not
+        // really interested in available peers.
+        if !self.bootstrap_node_vms.is_empty() {
```

Comment:
People can run a `new` deployment with 0 bootstrap nodes (testing)? In that case,it would still be useful to know the genesis multiaddr. We should probably just move the `genesis_mutliaddr` print outside this check.